README.md

```diff
@@ -40,7 +40,7 @@ To run this:
 
 The tool will download raw data from all feed end points and store it in the `raw_data` table.
 
-It will only download new data when re-run. It stores state of how far it got in `raw_next_url` column of `publisher_feed` table. 
```

Comment:
Stripping spaces thing; is that deliberate or something your editor does automatically? I think before we had something where your editor did this automatically and we tried to turn it off as it might cause problems.